symplyphysics/laws/electricity/power_from_energy_time.py

```diff
@@ -12,6 +13,17 @@
 # TODO: this law should be derived from definition power_is_energy_derivative
 power, energy, time = symbols('power energy time')
 law = Eq(power, energy / time)
+constant_1, constant_2 = symbols('constant_1 constant_2')
```

Comment:
Might be a good idea to hint what are these constants. I guess constant_1 is constant_power and constant_2 is initial energy.